package.json

```
@@ -0,0 +1,18 @@
+{
+  "name": "angular-buch-cdn",
+  "version": "1.0.0",
+  "description": "Angular Buch Content Delivery Network (CDN)",
+  "type": "module",
+  "devDependencies": {
+    "sharp": "^0.33.2",
+    "husky": "^9.0.11"
+  },
+  "scripts": {
+    "start": "node generate-images",
```

`npm start` irritiert mich, weil es eigentlich einen Build ausführt.
Was hältst du davon, es umzubenennen?


```suggestion
    "generate-images": "node generate-images",
```


oder einfach nur

```suggestion
    "build": "node generate-images",
```